Composer autoload error corrected!

neetikaaryal opened the pull request:
There was a conflict with composer overload which was corrected in line 30.

neetikaaryal committed:
Composer autoload error corrected!

There was a conflict with composer overload which was corrected in line 30. 
"require_once 'vendor/autoload.php';" corrected to "require_once __DIR__ . ‘/vendor/autoload.php’;"

sanzeeb3 commented on come-back.php:
```diff
@@ -27,7 +27,7 @@
 const COME_BACK_VERSION = '1.3.4';
 
-require_once 'vendor/autoload.php';
+require_once __DIR__ . ‘/vendor/autoload.php’;
```
The character `‘` is actually invalid. This should be replaced with `'`.

> The character U+2018 "‘" could be confused with the character U+0060 "`", which is more common in source code

neetikaaryal replied:
Hey @sanzeeb3 , I have corrected the issue. Let me know if it's good now.

sanzeeb3 reviewed (changes requested):
Hey @neetikaaryal , thanks for the PR. There's one thing to note. See the review:

neetikaaryal committed:
Issue resolved.

I have resolved the issue you mentioned. Let me know if you notice other issues.
